readme.md

```diff
@@ -30,6 +30,8 @@ console.log(link);
 
 For unsupported terminals, the link will be printed in parens after the text: `My website (https://sindresorhus.com)`.
 
+Use `terminalLink.stderr(text, url, [options])` for outputting to stderr.
```

Comment:
This makes it sound like the function call outputs to stderr, which it doesn't.

Reply:
How about "Use [...] to create a link for use in stderr."?

Reply:
👍